app/models/container_project.rb

```diff
@@ -71,7 +71,7 @@ def perf_rollup_parents(interval_name = nil)
   end
 
   def aggregate_memory(targets = nil)
-    Hardware.where(:computer_system => computer_systems).sum(:memory_mb)
+    Hardware.where(:computer_system => computer_systems).sum(:ram_size_in_bytes)
```

Comment:
this probably needs to return the same units as before

so divide (and to_i) to get answer?